materials/models/player/boss/yeti/yeti_face_invun.vmt

```diff
@@ -0,0 +1,128 @@
+// Modified to use heavy's stuff by Kirillian,, custom invulnerability textures by Kirillian.
```

Comment:
double `,` after `Kirillian`. Also calling name twice? I expect words to be along something like this:
```
Modified to use as heavy's custom model, and custom invulnerability textures made by Kirillian.
```

Also adding name and link makes me think others need to ask for permission, which that is not true as Kirillan is fine with it. Worth adding that everyone is allowed to use without permission